control-plane/stor-port/src/types/v0/transport/volume.rs

```diff
@@ -76,23 +76,41 @@ pub struct VolumeState {
 pub struct VolumeUsage {
     /// Capacity of the volume in bytes.
     capacity: u64,
-    /// Allocated size in bytes, related to a single healthy replica.
+    /// Allocated size in bytes, related the largest healthy replica, including snapshots.
     /// For example, if a volume has 2 replicas, each with 1MiB allocated space, then
     /// this field will be 1MiB.
     allocated: u64,
-    /// Allocated size in bytes, accrued from all the replica.
-    /// For example, if a volume has 2 replicas, each with 1MiB allocated space, then
-    /// this field will be 2MiB.
+    /// Allocated size in bytes, related the largest healthy replica, excluding snapshots.
+    allocated_replica: u64,
+    /// Allocated size in bytes, related the largest healthy replica's snapshots.
```

Comment:
Is it largest healthy replica's snapshot or largest snapshot of a that replica?

Reply:
In code it's the snapshot of the largest healthy replica, but maybe should be the largest snapshot among all healthy replica, wdyt?

Reply:
Yeah, not sure. The current one is fine probably?